internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -251,44 +267,46 @@ func (opts *Opts) setTier() {
 	}
 }
 
-func (opts *Opts) defaultValues() error {
-	if !opts.defaultValue {
-		return nil
-	}
-
-	opts.SkipSampleData = true
-	opts.SkipMongosh = true
+func (opts *Opts) fillDefaultValues(values *DefaultValues) error {
```

Comment:
this reads like a bit of a constructor, let's make it one 
```suggestion
func (opts *Opts) newDefaultValues() (*DefaultValues, error) {
  values := &DefaultValues{}
  ...
  return values, nil
```

Reply:
great! updated, thank you.